app/src/main/java/com/github/epfl/meili/poi/PoiInfoFragment.kt

```diff
@@ -1,30 +1,47 @@
 package com.github.epfl.meili.poi
 
+import android.Manifest
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
 import android.os.Bundle
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
+import android.view.View.GONE
+import android.view.View.VISIBLE
 import android.view.ViewGroup
+import android.widget.Button
 import android.widget.ImageView
 import android.widget.TextView
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
+import com.github.epfl.meili.MainApplication
 import com.github.epfl.meili.R
 import com.github.epfl.meili.models.PointOfInterest
 import com.google.android.libraries.places.api.model.Place
 import com.google.android.libraries.places.api.net.*
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import io.github.ponnamkarthik.richlinkpreview.RichLinkViewSkype
+import io.github.ponnamkarthik.richlinkpreview.ViewListener
 
 /**
  * Fragment to be displayed inside of PoiActivity and which contains basic info about POI
  */
-class PoiInfoFragment(val poi: PointOfInterest) : Fragment() {
+class PoiInfoFragment(val poi: PointOfInterest) : Fragment() { //TODO verify that everything working properly when better connection
```

Comment:
Is this a leftover TODO or something we should do in the future?

Reply:
Leftover thanks!